### PR TITLE
JBDS-4320 Show Hyper-V even if it is not detected with link to documentation how to configure and use it with CDK 3

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

**Comment by sudhirverma** [comment]
test this

**Comment by sudhirverma** [comment]
test this

File: browser/model/hyperv.js [hunk 1]
```diff
@@ -15,10 +15,11 @@ class HypervInstall extends InstallableItem {
   detectExistingInstall() {
     if (Platform.OS == 'win32') {
       return Platform.isHypervisorEnabled().then((detected)=>{
-        if(detected) {
+        if(detected || detected === false) {
```
Comment (dgolovin) [inline browser/model/hyperv.js]: new condition is not needed here.

File: browser/model/hyperv.js [hunk 2]
```diff
@@ -15,10 +15,11 @@ class HypervInstall extends InstallableItem {
...
           this.addOption('detected', '', '', detected);
         }
         this.selectedOption = 'detected';
+        this.hypervdetection = detected;
```
Comment (dgolovin) [inline browser/model/hyperv.js]: not needed as well.

File: browser/pages/confirm/confirm.html [hunk 3]
```diff
@@ -281,16 +281,28 @@
...
           </div>
-          <span id="hyperv-name" class="product-name">{{checkboxModel.hyperv.productName}}</span><span id="hyperv-version" class="product-version">{{checkboxModel.hyperv.option['detected'].version}}</span>
+          <span id="hyperv-name" class="product-name">{{checkboxModel.hyperv.productName}}</span>
+          <span id="hyperv-version" class="product-version">{{checkboxModel.hyperv.option['detected'].version}}</span>
           <div id="hyperv-description">{{checkboxModel.hyperv.productDesc}}</div>
         </div>
       </div>
     </div>
 
+    <div class="help-block" ng-show="checkboxModel.hyperv.hasOption('detected') && !checkboxModel.hyperv.hypervdetection">
```
Comment (dgolovin) [inline browser/pages/confirm/confirm.html]: Should be visible on Windows only

File: browser/pages/confirm/confirm.html [hunk 4]
```diff
@@ -281,16 +281,28 @@
         </div>
         <div class="product-container">
           <div class="warnings-in-panel">
-            <span id="hyperv-installed-note" class="installed-note" ng-show="checkboxModel.hyperv.hasOption('detected')">
+            <span id="hyperv-installed-note" class="installed-note" ng-show="checkboxModel.hyperv.hasOption('detected') && checkboxModel.hyperv.hypervdetection">
               <i class="fa fa-check"></i> Using detected version {{checkboxModel.hyperv.option.detected.version}}
             </span>
+            <span id="hyperv-installed-note" class="installed-note" ng-show="checkboxModel.hyperv.hasOption('detected') && !checkboxModel.hyperv.hypervdetection">
```
Comment (dgolovin) [inline browser/pages/confirm/confirm.html]: Should be visible when not detected

**Comment by dgolovin** [comment]
Should be something like this in the end
hyperv-not-detected

**Comment by jrichter1** [comment]
I think it should at least only show on systems that actually support Hyper-v (meaning Win 8/10 Pro and higher) - the systems where this command
https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/services/platform.js#L104 actually returns something. 
It looks weird when it shows up on my Win10 home and the first step is to check if I have Pro/Enterprise edition.

File: browser/pages/confirm/confirm.html [hunk 5]
```diff
@@ -284,13 +284,25 @@
...
+          <span id="hyperv-version" class="product-version">{{checkboxModel.hyperv.option['detected'].version}}</span>
           <div id="hyperv-description">{{checkboxModel.hyperv.productDesc}}</div>
         </div>
       </div>
     </div>
 
+    <div class="help-block" ng-show="checkboxModel.hyperv !== undefined && !checkboxModel.hyperv.hasOption('detected')">
+      <span class="pficon pficon-info"></span>
+      <span>Follow
+        <a type="button" class="pointer" ng-click="confCtrl.download('https://access.redhat.com/documentation/en-us/red_hat_container_development_kit/2.4/html/installation_guide/installing_container_development_kit_on_microsoft_windows#windows-add-sw')">this steps</a>
```
Comment (jrichter1) [inline browser/pages/confirm/confirm.html]: these* steps - gotta uphold the grammar :)

**Pull request merged**